installers/ansible/roles/pgo-operator/templates/pgo-client.json.j2

```diff
@@ -24,6 +24,11 @@
                 }
             },
             "spec": {
+                "imagePullSecrets": [
+{% if pgo_image_pull_secret %}
+                    { "name": "{{ pgo_image_pull_secret }}" }
+{% endif %}
+                ],
```

Comment:
Two things:

1. Why not wrap the whole `{% if %}` block around the `"imagePullSecrets"` section? It seems like it's either we have it, or we don't.

1. I'd recommend aligning the `{% if %}` statement with that of the text block.  I think it's easier to read, and typically how I've written/seen Jinja2 style templates.

Reply:
> Two things:
> 
> 1. Why not wrap the whole `{% if %}` block around the `"imagePullSecrets"` section? It seems like it's either we have it, or we don't.

It's like this in other places, probably because we have more than one image/secret sometimes. Not here, though. Done.

> 2. I'd recommend aligning the `{% if %}` statement with that of the text block.  I think it's easier to read, and typically how I've written/seen Jinja2 style templates.

Done.